src/i18n/index.ts

```
@@ -54,6 +54,225 @@ const resources = {
+      lastMonth: 'Last Month',
+      lastYear: 'Last Year',
+      retrying: 'Retrying...',
+      
+      // Loyalty Card
+      availableRewards: 'Available Rewards!',
+      viewCard: 'View Card',
+      allRewardsEarned: 'All rewards earned!',
+      noRewardsAvailable: 'No rewards available',
+      
+      // Loyalty Card Additional
+      business: 'Business',
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)